website_blog_mgmt/tests/test_website_blog_flow.py

```diff
@@ -32,23 +32,18 @@ def setUp(self):
         self.blog_blog_obj = self.env['blog.blog']
         self.blog_post_obj = self.env['blog.post']
         # Create a new blog
-        self.test_blog = self.blog_blog_obj.sudo(
-            self.user_blogmanager).create({
-                'name': 'New Blog',
-                'description': 'Presentation of new Odoo features'
-            })
+        self.test_blog = self.blog_blog_obj.sudo(self.user_blogmanager).create(
+            {'name': 'New Blog',
+             'description': 'Presentation of new Odoo features'
+             })
         # Create a first post
         self.test_blog_post_1 = self.blog_post_obj.sudo(
-            self.user_blogmanager).create({
-                'name': 'New Post 1',
-                'blog_id': self.test_blog.id,
-            })
+            self.user_blogmanager).create({'name': 'New Post 1',
```

Comment:
I agree, although sometimes the style use by the first contributor is not the one of my choice (which in this case, it is), I prefer not to change it because it creates larger diff and add a lot of noise on the needed changes to make this work.